Meta/run.sh

```diff
@@ -30,9 +30,13 @@ fi
 
 # To support virtualization acceleration on mac
 # we need to use 64-bit qemu
-if [ "$(uname)" = "Darwin" ] && [ "$(uname -m)" = "x86_64" ]; then
+if [ "$(uname)" = "Darwin" ]; then
 
-    [ -z "$SERENITY_QEMU_BIN" ] && SERENITY_QEMU_BIN="qemu-system-x86_64"
+    if [ "$SERENITY_ARCH" != "aarch64" ]; then
+        [ -z "$SERENITY_QEMU_BIN" ] && SERENITY_QEMU_BIN="qemu-system-x86_64"
+    else
+        SERENITY_QEMU_BIN="qemu-system-aarch64"
```

Comment:
I think this breaks providing a custom path for `SERENITY_QEMU_BIN` on macOS. Should be handled automatically starting from line 56.

Reply:
i.e. only having the `if` as is without an `else` should be enough.

Reply:
You're right.  I added -z "$SERENITIY_QEMU_BIN" to the aarch64 path.